src/main/java/orion/exceptions/OrionException.java

```diff
@@ -0,0 +1,21 @@
+package orion.exceptions;
+
+/**
+ * Represents a base exception class for all exceptions related to the Orion application.
+ * <p>
+ * This abstract class extends {@link Exception} and is used as a base class for
+ * custom exceptions in the Orion application. It allows for the creation of specific
+ * exception types with custom error messages.
+ * </p>
+ */
```

Comment:
I like the javadocs for the abstract OrionException class! Concise, descriptive, and tells you its purpose.